docs/samples/Microsoft.ML.Samples.StaticPipe/DatasetCreator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.ML.Samples.StaticPipe
+{
+    public static class DatasetCreator
```

Comment:
>DatasetCreator [](start = 24, length = 14)

@shauheen, @galoshri  What do we want to do with the datasets for the samples? This is not a solution, as this function won't display/be referenceable by the samples. 

Shall we create a small nuget package with 6 functions that generate those datasets, and import that in our samples?
scikit examples always import from the datasets package. 

Shall we package/re-distribute the datasets we are using for testing?(i bet we'd want to stay out of the legal work for that.)